### DISCUSSION TITLE
Dual GPU with ROCm on Windows is SLOWWW

### DISCUSSION THREAD

**Discussion by ccbadd** [ROCm/ROCm#2491] [General]
I have a pair of w6800s in my window 11 PC running ROCm5.5.  When I run an LLM with llama.cpp or Koboldcpp they crash unless I use the low vram option.  This causes the v and k cache's not to be offloaded to the GPUs and the model run almost as slow as just running it on the cpu.  I do not see this issue running the same setup, only with ROCm 5.6 under Ubuntu.  The same setup with NVidia 3090's are nearly 20 t/s and I'm getting 4!  Any idea of the problem and maybe how to fix it?

**Comment by fluidnumericsJoe** [comment]
Without profiles showing what is happening with the code, can't really help.. run the code under rocprof using

```
rocprof --sys-trace ./application
```
To get a trace profile. You can visualize the profile by loading the `results.json` output by the profile to https://ui.perfetto.dev

I'm going to resist the urge to tell you not to use windows when you know it works on Linux, but why are you making your life harder with this choice?
